.inventory-api.yaml

```diff
@@ -6,6 +6,11 @@ server:
     address: localhost:9000
 authn:
    allow-unauthenticated: true
+#   oidc:
+#     authn-server-url: http://localhost:8084/realms/redhat-external
+#     skip-client-id-check: true
+#     skip-issuer-check: true
+#     principal-user-domain: localhost
```

Comment:
Is this comited on purpose? should we remove if it is no longer relevant ?

Reply:
Yes, I add on purpose. For devs who want to run a local keycloak and test oidc flow